.github/workflows/ci-ros.yml

```diff
@@ -0,0 +1,64 @@
+name: Crazyswarm ROS CI
+
+on: [push, pull_request]
+
+env:
+  ROS_CI_DESKTOP: "`lsb_release -cs`"  # gives Ubuntu name, e.g. [precise|trusty|...]
+  ROS_PARALLEL_JOBS: "-j8 -l6"
+
+jobs:
+  build:
+
+    strategy:
+      matrix:
+        include:
+          - os: ubuntu-16.04
+            ros: kinetic
+          - os: ubuntu-18.04
+            ros: melodic
+
+    runs-on: ${{ matrix.os }}
+
+    steps:
+
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: Install Dependencies
+      run: |
+        # Install non-ROS system dependencies.
+        sudo apt install -y swig libpython-dev python-numpy python-yaml python-matplotlib python-pytest
+        # Install firmware cross-compiler and hardware driver dependencies.
+        sudo add-apt-repository -y ppa:team-gcc-arm-embedded/ppa
+        sudo apt-get update
+        sudo apt install -y gcc-arm-embedded libusb-1.0-0-dev sdcc
+        # Set the python path manually to include /usr/-/python2.7/dist-packages
+        # as this is where apt-get installs python packages.
+        PYTHONPATH=$PYTHONPATH:/usr/lib/python2.7/dist-packages:/usr/local/lib/python2.7/dist-packages
```

Comment:
Kind of odd that this is needed?

Reply:
Deleted, wasn't needed. Not sure why it was there to begin with.